### DISCUSSION TITLE
Version controlled schema dumps

### DISCUSSION THREAD

**Discussion by Qwertie-** [SeaQL/sea-orm#810] [Q&A]
Looking to find out if schema files / dumps is something integrated in to SeaORM. Something similar to the ActiveRecord schema.rb.

Just trying to work out generally what the recommended way to load a new database from scratch is rather than running all migrations.

**Comment by billy1624** [comment]
Hey @Qwertie-, In SeaORM we don't have a schema dump. I'm not sure what you mean by loading a new database from scratch. Are you deploying the same app in multiple instance? Each of it operate independently and with its own set of data (stored on database) but sharing the same code base?

**Reply by Qwertie-** [reply]
Hey, sort of. I expect to only have one production instance but many developers who will need to get set up and create a db from scratch. With Rails I'd run a command that creates a schema matching a schema dump which is kept in the repo. What would be the standard way to set up a new db using SeaORM? Running all migrations?

**Comment by tyt2y3** [comment]
Actually, sea-schema outputs sea-query schema statements as it's API, so it's very much a mysqldump. Not tried using it this way though. I guess need to wire it up using a main.rs
